### ISSUE TITLE
Link to cross compile libraries is bunk

### ISSUE THREAD

**Issue by Barugon** [gtk-rs/gtk-rs#120]
In the tutorial for [cross compiling Windows from Linux](https://gtk-rs.org/docs-src/tutorial/cross), there is a link to download the mingw-w64-gtk3 libraries from win32builder.gnome.org. However, that site does not exist. Is there another place to get the libraries for distributions other than Arch?

**Comment by sdroege** [comment]
We should just link to https://www.gtk.org/docs/installations/windows/

**Comment by anieuwland** [comment]
@sdroege I agree. I did that in my PR [here](https://github.com/gtk-rs/gtk-rs.github.io/pull/181).

However, because I did do not find GTK Windows installations instructions particularly friendly, I also kept the instructions for Arch Linux and added those for Fedora. They make it much easier to achieve cross-compilation.

I am curious to hear whether you agree or not and find the PR acceptable.
